src/components/input-dropdown/input-dropdown.component.spec.ts

```diff
@@ -206,4 +212,40 @@ describe('InputDropdownComponent', () => {
         expect(openChangeSpy).toHaveBeenCalledTimes(2);
         expect(openChangeSpy).toHaveBeenCalledWith(false);
     });
+
+    it('should clear the value when allowNull is set to true', async () => {
+        component.allowNull = true;
+        component.dropdownOpen = true;
+
+        component.selected = 'One';
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        const clearButton = nativeElement.querySelector('.ux-select-icon.ux-select-clear-icon') as HTMLButtonElement;
+        clearButton.click();
+
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        const title = nativeElement.querySelector<HTMLHeadingElement>('.selection');
```

Comment:
The type of this element object is not HTMLHeadingElement. HTMLElement is probably fine.